Sources/BraveWallet/Crypto/AssetIconView.swift

```diff
@@ -136,13 +136,18 @@ struct NFTIconView: View {
   var shouldShowNetworkIcon: Bool = false
   
   @ScaledMetric var length: CGFloat = 40
+  var maxLength: CGFloat?
   @ScaledMetric var tokenLogoLength: CGFloat = 15
+  var maxTokenLogoLength: CGFloat?
   
   @ViewBuilder private var tokenLogo: some View {
     if shouldShowNetworkIcon, let image = network.nativeTokenLogoImage {
       Image(uiImage: image)
         .resizable()
-        .frame(width: 15, height: 15)
+        .frame(
+          width: min(tokenLogoLength, maxTokenLogoLength ?? tokenLogoLength),
+          height: min(tokenLogoLength, maxTokenLogoLength ?? tokenLogoLength)
+        )
```

Comment:
can we also add an overlay of a white circle as in `tokenLogo`?
![Simulator Screenshot - iPhone SE (3rd generation) - 2023-06-07 at 13 01 39](https://github.com/brave/brave-ios/assets/1187676/f4bc8665-ac96-4dd2-991d-c8dac73f8747)